### PR TITLE
BaSP-MR-70: Add and Edit Form in Super Admin

### PR DESCRIPTION
**Pull request by IvanJukonis**
Hello, I have submitted the PR for ticket number 70. This ticket required the development of the Update and Create functionalities for Super Admins, displaying a modal to confirm the modification. Additionally, I had to add the functionality of showing the Creation form, but enabled for performing updates.

All the requirements of the ticket have been resolved.

Due to the List and Delete functionalities created in ticket 69, which were also under my responsibility, not being merged, they were implemented in the first commit in order to develop this PR. By submitting both PRs, I will be able to resolve the conflicts easily.

Functionalities to add: The form was not styled due to lack of time.

### CODE REVIEW

**Comment by IvanJukonis** [comment]
[SUPER ADMIN - Add/edit form](https://trello.com/c/kiWU79Cd/70-super-admin-add-edit-form)

File: src/Components/SuperAdmins/index.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        console.error('Failed to add super admin');
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const updateItem = async (updatedItem) => {
+    try {
+      // eslint-disable-next-line no-unused-vars
```
Comment (FrancoLelli) [inline src/Components/SuperAdmins/index.jsx:74]:
```suggestion
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
I think you should delete the comment, but, its ok!! Nice Job!

File: src/Components/SuperAdmins/index.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      const { _id, __v, ...updatedData } = updatedItem;
+      console.log(JSON.stringify(updatedData));
```
Comment (CristianLotorto) [inline src/Components/SuperAdmins/index.jsx:76]: try to delete console.log

File: src/Components/SuperAdmins/index.js
Comment (NicoCagnina) [inline src/Components/SuperAdmins/index.js:1]: why did you remove this file?

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.js
Comment (IvanJukonis) [reply src/Components/SuperAdmins/index.js:1]: I changed the file index.js to index.jsx in order to follow the instructions of the weekly class. Im not sure why I deleted it, on my other PR I just changed the termination.

File: src/Components/SuperAdmins/index.jsx (hunk 2 above)
Comment (IvanJukonis) [reply src/Components/SuperAdmins/index.jsx:76]: Same as responded to Franco.

**Review by FrancoLelli — APPROVED** [review]

File: package.json
Comment (cbaleani) [inline package.json:1]: The package.json should not be changed.

File: package-lock.json
Comment (cbaleani) [inline package-lock.json:1]: Changes in the package-lock.json shouldn't be pushed

File: src/Components/SuperAdmins/Form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import ModalsConfirmation from '../../Modals/ModalConfirm';
```
Comment (FrancoLelli) [inline src/Components/SuperAdmins/Form/index.jsx:2]:
```suggestion
import ModalConfirm from '../../Modals/ModalConfirm';
```

File: src/Components/SuperAdmins/Form/index.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          Agregar Super Admin
+        </button>
+      )}
+      {showBtnMod && (
+        <button type="submit" onClick={() => setActionType('modify')}>
+          Modificar Super Admin
+        </button>
+      )}
+      {modalModifyConfirmOpen && (
+        <ModalsConfirmation
```
Comment (FrancoLelli) [inline src/Components/SuperAdmins/Form/index.jsx:69]:
```suggestion
        <ModalConfirm
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Change names, please!!

**Review by FrancoLelli — APPROVED** [review]

**Review by cbaleani — APPROVED** [review]
Looks good!

**Comment by GleViDe** [comment]
### 2nd Testing Results
Ivan please check the following bug report. For more details, go to the ticket 'MR-114' in Trello: [MR-114](https://trello.com/c/KywYrFTi/114-super-admin-cant-add-new-super-admin)

Tested locally on https://nico-megarocket-app-git-feature-mr-70superadmin-add-43525f-basp.vercel.app/super-admins
Test result: Failed

Bug found:
After filling out the form to add a new Super Admin and left clicking on the "Agregar Super Admin" button, nothing happens.

**Pull request merged**